src/test/resources/outputs/redshift/basics/exclude.sql

```diff
@@ -1,115 +1,115 @@
---#[exclude-00]
```

Comment:
self-review: commented-out the file for now. Will tackle exclude transpilation in an upcoming PR